test.js

```diff
@@ -0,0 +1,61 @@
+// //1
+// class Human {
+//   constructor(name, age) {
+//     this.name = name;
+//     this.age = age;
+//   };
+//
+//   ageOneYear(str) {
+//     this.age++;
+//     return `It's my birthday! Now, I'm ${this.age}`;
+//   };
+//
+//   eating() {
+//     return `Mmmm, mmmm.... I'mmmm loving itttt!`;
```

Comment:
We want to log the line to the console, so that when we call upon the function `eating`, it will automatically console.log the line rather than returning it:
```
console.log("Mmmm, mmmm.... I'mmmm loving itttt!")
oliver.eating() // This will log the line without needing to be wrapped in a console.log statement!
```
Please let me know if you have any questions or if I'm being unclear~